### PR TITLE
Nickakhmetov/HMP-215 Add tooltips to publication page

### PR DESCRIPTION
**Pull request by NickAkhmetov**
This PR adds explanatory tooltips to the publication page. I used my other PR as the base branch to avoid conflicts since both edit the publication summary.
 

https://github.com/hubmapconsortium/portal-ui/assets/19957804/6aaecd52-e182-476b-a226-9e25d4fb49a9

### CODE REVIEW

**Review by john-conroy — APPROVED** [review]
Looks good, thanks!

File: context/app/static/js/pages/Publication/Publication.jsx [hunk 1]
```diff
@@ -62,8 +62,18 @@ function Publication({ publication, vignette_json }) {
         <PublicationsVisualizationSection vignette_json={vignette_json} uuid={uuid} />
       )}
       <Files files={files} uuid={uuid} hubmap_id={hubmap_id} />
-      <ContributorsTable contributors={contributors} title="Authors" />
-      {shouldDisplaySection.provenance && <ProvSection uuid={uuid} assayMetadata={publication} />}
+      <ContributorsTable
+        contributors={contributors}
+        title="Authors"
+        iconTooltipText="The authors are listed in the same sequence as the author list of the publication."
```
Comment (tsliaw) [inline context/app/static/js/pages/Publication/Publication.jsx:68]: Can you check if this tooltip is actually true? Is the author list in that table in the same order as the publication's author list? If it isn't, remove this tooltip.

Comment (lchoy) [reply context/app/static/js/pages/Publication/Publication.jsx:68]: For this one: https://portal-prod.stage.hubmapconsortium.org/browse/publication/72cbeb8ff605fd5017cb2666cd19dfb7, the order is mostly identical, but not exactly.

Comment (NickAkhmetov) [reply context/app/static/js/pages/Publication/Publication.jsx:68]: @tsliaw @lchoy The list in that publication matches the order in `contributors.tsv` in Globus as far as I can tell. We could specify that `The authors are listed in the sequence specified by the publication's uploaders` or something along those lines, but I'm not sure that information is as useful/applicable to viewers as it is to the uploaders themselves

Comment (tsliaw) [reply context/app/static/js/pages/Publication/Publication.jsx:68]: Let's remove this tooltip, if there's any confusion in the future about the ordering of this author list, we can add back this tooltip for clarification.

Comment (NickAkhmetov) [reply context/app/static/js/pages/Publication/Publication.jsx]: 👍 Works for me! The tooltip has been removed.

**Review by tsliaw — APPROVED** [review]

**Pull request merged**